engines/stark/gfx/renderentry.cpp

```diff
@@ -194,10 +194,16 @@ bool RenderEntry::intersectRay(const Math::Ray &ray) const {
 }
 
 VisualImageXMG *RenderEntry::getImage() const {
+	if (!_visual) {
+		return nullptr;
+	}
 	return _visual->get<VisualImageXMG>();
 }
 
 VisualText *RenderEntry::getText() const {
+	if (!_visual) {
+		return nullptr;
```

Comment:
In the implementation of `GameWindow::onScreenChanged()` I came across with render entries with no `Visual`. It feels strange to me since a `RenderEntry` without a `Visual` seems pretty useless. But anyway it needs to be handled.